cypress.config.ts

```diff
@@ -0,0 +1,15 @@
+import { defineConfig } from "cypress";
+
+export default defineConfig({
+  e2e: {
+    baseUrl:
+      // process.env.REACT_APP_ENV === "prod"
```

Comment:
Why is this commented out? Should we add the `REACT_APP_ENV` variable to .env set as `prod` and add another `.env.local` file for local use which has it set to `local`.  The `.env.local` file takes precedence over the `.env` file, but only exists locally and should be ignored by git.

Reply:
That is not necessary. I was trying the tests out for prod website manually and forgot to remove the comment